### PR TITLE
ISSD-1508 [marketplace] Create Table component

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.status-cell {
+	&-icon {
+		height: 0.5rem;
+		width: 0.5rem;
+	}
+}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.scss:6]:
```suggestion
.status-cell-icon {
	height: 0.5rem;
	width: 0.5rem;
}
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+};
+
+export default function StatusCell({
+	children,
+	className,
+	icon,
+	iconClass,
+}: StatusCellProps) {
+	return (
+		<div className={`align-itens-center d-flex status-cell ${className}`}>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.tsx:25]: use classNames()

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+			<div className="align-items-center d-flex">
+				<ClayIcon
+					className={`mr-2 status-cell-icon ${iconClass}`}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.tsx:28]: use classNames

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	className?: string;
+	icon: string;
+	iconClass?: string;
+};
+
+export default function StatusCell({
+	children,
+	className,
+	icon,
+	iconClass,
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/StatusCell/StatusCell.tsx:22]: iconClassName

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**